File: Programming Basics/JavaScript/Introduction/Labs/projects-creation.js
function solve(input) {
    const architectName = input[0];
    const projectsCount = Number(input[1]);

    const requiredHours = projectsCount * 3;
    console.log(`The architect ${architectName} will need ${requiredHours} hours to complete ${projectsCount} project/s.`);
}

File: Programming Basics/C++/Introduction/Labs/ProjectsCreation/ProjectsCreation.cpp
#include <iostream>
#include <string>
using namespace std;

int main()
{
    string architectName;
    int projectsCount;
    cin >> architectName >> projectsCount;

    int requiredHours = projectsCount * 3;
    cout << "The architect " << architectName << " will need " << requiredHours << " hours to complete " << projectsCount << " project/s." << endl;
}
